### ISSUE TITLE
Singing Voice Conversion (SVC) ,训练的数据量

### ISSUE THREAD

**Issue by wlf0322** [Plachtaa/seed-vc#122]
请问一下训练svc用了多少数据量，以及数据类型，是speech还是歌曲训练的。论文里面写到了vc的，似乎没有看到关于svc的数据，期待您的回复

**Comment by Plachtaa** [comment]
almost the same

**Comment by wlf0322** [comment]
> almost the same

Thank you for your reply. What you mean is that training svc also uses speech data, not song data, right? So, is svc retrained or fine-tuned on the vc model?

**Comment by Liangzheng-ZL** [comment]
Hi, I'm interested in this question as well, I noticed that the paper uses m4singer as training data, but in the answer [https://github.com/Plachtaa/seed-vc/issues/55#issuecomment-2501847071](url) it mentions that m4singer is only a part of it.  What
 is the amount of data used to train SVC, and does it include speech data?

**Comment by caixxiong** [comment]
I saw the answer at #93 

<img width="1862" height="990" alt="Image" src="https://github.com/user-attachments/assets/57d9cc18-4571-485d-8717-44391c295cc7" />
